lib/cli.rb

```diff
@@ -1,19 +1,25 @@
-class Cli
+require_relative './content.rb'
+require_relative './game.rb'
+
+class Cli 
   def call
     clear_screen
-    puts menu
-    puts select_marker
-  end
-
-  def menu
-    "Welcome to Tic Tac Toe!\n"
+    display_text(Content.welcome)
+    display_text(Content.select_marker)
+    user_input 
   end
 
-  def select_marker
-    "Choose 'X' or 'O' to play game: \n"
-  end
+  def display_text(text)
+    puts text + "\n\n"
```

Comment:
you can interpolate with "#{text}\n\n"

Reply:
commit 240bf9f926484a874279542f5ac97c443441f1f6